### PR TITLE
podspec file for cocoapods integration

### PR DESCRIPTION
**Pull request by ehuynh**
Podspec file to allow for integration of Scenarios using cocoapods #21.

### CODE REVIEW

File: Scenarios.podspec [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Pod::Spec.new do |s|
+  s.name     = 'Scenarios'
+  s.version  = '0.2.2'
+  s.license  = { :type => 'Apache 2.0', :file => 'LICENSE.txt' }
+  s.summary  = 'Human-oriented testing in Swift.'
+  s.homepage = 'https://github.com/outware/Scenarios'
+  s.authors  = { 'Adam Sharplet' => 'adsharp@me.com' }
```
Comment (sharplet) [inline Scenarios.podspec]: Heh, nice one. This should probably be "Adam Sharp" ;)

File: Scenarios.podspec [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+  s.source   = { :git => 'https://github.com/outware/Scenarios.git', :tag => "v#{s.version}" }
+  s.source_files = 'Scenarios/**/*.swift'
+
+  s.ios.deployment_target = '8.0'
```
Comment (sharplet) [inline Scenarios.podspec:10]: I'd prefer to get rid of the extra lines of whitespace.

**Comment by sharplet** [comment]
Ok this is great, and lints ok on my machine. However I'd like to make things a little more dynamic, if possible—an approach I've been taking on Regex.

We can dynamically compute these attributes:
- `name`: by stripping the `.podspec` extension from the current file name
- `tag`: by asking git to give us the current tag details
- `summary`: by querying the GitHub API

Would you mind taking a look at [STRegex.podspec#L2-4](https://github.com/sharplet/Regex/blob/6329c42929f873e33042b078a6a56672aa195250/STRegex.podspec#L2-L4) and incorporating those changes?

**Comment by ehuynh** [comment]
sure. i'll take a look.

File: Scenarios.podspec (hunk 1 above)
Comment (ehuynh) [inline Scenarios.podspec]: haha ... didn't realise i did that

**Comment by gavanchan-outware** [comment]
New updates to Cocoapods and this no longer works. Apologies for the long delay. 

Shall we pick this up and continue with an attempt to get this up and running?

**Comment by ehuynh** [comment]
hey @gavanchan-outware, I haven't been using cocoapods these days so I'm not across the changes required to get this to work. In addition I haven't been using Scenarios either, so my motivation to get this working with cocoapods is not there. Apologies. You can close this PR and open a new issue if the need is still there.

**Comment by gavanchan-outware** [comment]
Absolutely, @ehuynh. The effort is well appreciated. Thanks for the contribution. 🙂

**Pull request closed without merging**